src/components/NodeSelectorPanel.tsx

```diff
@@ -153,21 +171,10 @@ const NodeSelector = ({ data, height }: NodeSelectorProps) => {
                         >
                             <Accordion
                                 allowMultiple
-                                defaultIndex={data.map((item, index) => index)}
+                                defaultIndex={schemata.schemata.map((item, index) => index)}
                             >
-                                {data.map(({ category, nodes: categoryNodes }) => {
-                                    const matchingNodes = matchesSearchQuery(category)
-                                        ? categoryNodes
-                                        : categoryNodes.filter(
-                                              (n) =>
-                                                  matchesSearchQuery(`${category} ${n.name}`) ||
-                                                  matchesSearchQuery(`${n.subcategory} ${n.name}`)
-                                          );
-
-                                    // don't show categories without nodes
-                                    if (matchingNodes.length === 0) return null;
-
-                                    const namespaceMap = getNamespaces(matchingNodes);
+                                {[...byCategory(matchingNodes)].map(([category, categoryNodes]) => {
+                                    const namespaceMap = getNamespaces(categoryNodes);
 
```

Comment:
This isn't anything to do with your changes, I just realized I left in the word namespaces here. This is a relic from before I added the subcategories, and used to have nodes grouped by the first part of their names, back when they were named like "IMAGE::LOAD". It should probably be renamed.

Reply:
Alright